ogc/features/url.go

```diff
@@ -195,53 +225,53 @@ func parseLimit(params url.Values, limitCfg engine.Limit) (int, error) {
 	return limit, err
 }
 
-func parseBbox(params url.Values) (*geom.Extent, int, error) {
-	bboxCrs, err := parseSRID(params, bboxCrsParam)
+func parseBbox(params url.Values) (*geom.Extent, SRID, error) {
+	bboxSRID, err := parseSRID(params, bboxCrsParam)
 	if err != nil {
-		return nil, -1, err
+		return nil, undefinedSRID, err
 	}
 
 	if params.Get(bboxParam) == "" {
-		return nil, bboxCrs, nil
+		return nil, undefinedSRID, nil
 	}
 	bboxValues := strings.Split(params.Get(bboxParam), ",")
 	if len(bboxValues) != 4 {
-		return nil, bboxCrs, fmt.Errorf("bbox should contain exactly 4 values " +
+		return nil, bboxSRID, fmt.Errorf("bbox should contain exactly 4 values " +
 			"separated by commas: minx,miny,maxx,maxy")
 	}
 
 	var extent geom.Extent
 	for i, v := range bboxValues {
 		extent[i], err = strconv.ParseFloat(v, 64)
 		if err != nil {
-			return nil, bboxCrs, fmt.Errorf("failed to parse value %s in bbox, error: %w", v, err)
+			return nil, bboxSRID, fmt.Errorf("failed to parse value %s in bbox, error: %w", v, err)
 		}
 	}
 
-	return &extent, bboxCrs, nil
+	return &extent, bboxSRID, nil
 }
 
-func parseSRID(params url.Values, paramName string) (int, error) {
-	srid := wgs84SRID
+func parseSRID(params url.Values, paramName string) (SRID, error) {
 	param := params.Get(paramName)
 	if param == "" {
-		return srid, nil
+		return undefinedSRID, nil
 	}
 	param = strings.TrimSpace(param)
 	if !strings.HasPrefix(param, crsURLPrefix) {
```

Comment:
moeten we ook URNs ondersteunen? bv `urn:ogc:def:crs:EPSG::28992`

Reply:
nee hoeft niet. In de OGC API spec wordt alleen ondersteuning voor URIs genoemd. Daarnaast is er (vanaf part3 pas) optioneel support voor CURIE (compact uri), dat is weer iets anders dan een URN.